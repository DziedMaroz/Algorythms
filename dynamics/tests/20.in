218087422623
553
34524721
1746808
7732
1614540872
1509
12047
55601333
85321955
9
481549
644167530
6151805333
9343
909656611
9291242291
6
55750476
7264320467
02
23723269
74803960180
7992
536656
8098
0571
94638247495
234033358
6985490128
6673406173
252
94065
42471017766
5439406
866105843687
268
51442796
41
85000330
240
51333087672
520
339968370002
546422
2095
78486072
41887413
74
847603682
8900424
1
90212
33082
8
646728494228
99247904539
191509
7187946
254595755799
808163925
345003
6766193000
609702
035485
8650197
172495772793
525
401841
364
601205291
79326452
19515
18
3215159
034457
336710092640
742781843977
388049569
6352
3
80259472
05
367222105268
1
668942355755
8
7870672564
62100590
0
179805
567582999
1914
36342
03608547
79577641485
0800186
64661057018
2167655
237491271
843973590
0198319
7639516
4194321
5858722590
05876
11839
56
6520638
542880846712
75
813711
57
72937879479
2317
3858
8
87964
4465860
113335977290
641120
64015885
3594326771
4461261372
87511
230517114
805351
869051906
1130
6
61531749
7817
90164881688
6878
830851
25978386
6631115
3
414
9354173003
036337
06753
2996455
57800
54
94
4742501831
85
498464
73569605438
23587977070
7402477182
113461
8
571
3939
1
211438912
63
5136
8
4809174865
7
7
298678561659
2865
366
793
2430
3
723168
1512926975
264431
558665071583
1280808
0469
836456623
9170626684
5
4957999
4540661
676
7
660050394
4618
8294324455
54728001
0
27205937117
1
961373067019
53552284577
870777485
6503
4032
22256098
81749153428
090520
385356510649
7
2941390891
6847
5
25443226
133949
54703957842
26563300450
47
5135367386
6178
4756610
167111
354476
7008
0487682762
345968001009
226989263174
9889019616
701516
63961416
05751836
4
1640539
461339
06490502505
7
52106541
954287488
148658868836
79
48
9848099
7
04724715
1428468
16
68
822069667055
286
3029
77773539311
88
5070
9
545603782521
02479396438
7281959808
68274932614
564
99
0
82304219202
8
3
757388504
44
5
0718369
841680482550
23177
06
39330648751
54870674
1
0486
66123385289
7980
010292738
299889377
954689435832
0
702534
7925060929
7773
319
1265
98
04
98662712
200356253966
80167130361
26027
72881848
89202378
48
28330602
200415904881
9126412283
2883070
5907
291
0820
8017
5990
044496
088
59
66572087085
031
4042
5947
39835
874271139233
24549895391
5757192548
705460112429
98710
014
6794
003867
9448
7449716
97781589789
274326223
212356363694
0
33944
20663302
0018492217
1997198539
64225330333
4357206
5064743041
4709
826262755932
596172
2125
64
0984750
250769245
24917321
0621
55870787662
149057178
907138
22023715
196762
08535
11010929219
21634359
675
4209395
761
973156747
23153324
99688621
144458584363
86680762083
26698430605
117942
41746
8085
08
27855398
3846
75737335
17742
439111423
9687827691
54970222
3638134
81531424033
478400411
149848952101
72841
618709
86630
40670752413
912289361
226457343
3267306500
51
389
715
74236
8208809
02674805396
42
80928
51495
9698
5922
6667516712
20158217
285514
5322471095
78075119
34397
3542699538
256947728849
697596770045
6260
3845
6700
11330
4783269
377
8931
8106890
700383
5
467
9135447583
8561119
3850
0185
589546582369
111502
29482098316
803319863
48
4082852095
80
269754
93681
896564
29694740832
0049469
80648904093
508
815557
57717
73
9858821
79818934800
129043680
82
6
789480749476
5061635432
366032614658
500306
1992519
9953895
75
02983899128
71929668
1129760778
22
2030456903
65824943372
05877
8079636
10
2720575
046328
459586031
80
5691897222
70566837662
5
08209651964
3028082259
2295607448
73
2358256
575459747
2350204790
745471316413
846
433628002234
5268335495
197655
399001391395
45486
820525451893
96
34829613083
06018782190
502191095086
115
265059601
715
1026034215
7
692056219190
32666811316
57496885
63298778
6201289925
1317156865
0065173
940696456025
2178
4015
5029702
3403972
27943
17283
6941
400302
01
8
3
96
504717596675
128227628193
69964170
110
3503290
4386664
39869826
59400255755
93059600534
2537966147
3529939621
8
528
671000931553
1
7246
359698
38656773
4711046
4797327
963
893373933
8562645262
9
24995547083
67491
0585508
29813810
156235559558
512072946588
28
62860066
5
291119383
2
595752595829
8714762178
7626
079325235181
40665488469
9530
61143609587
6220265505
787
143561
116833451326
2495621
183000029
5
3385
31914461
07877776659
051260
84324626
026662810693
0278480
4967568329
4
02236
2511351
0
