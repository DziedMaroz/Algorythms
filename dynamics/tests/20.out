8
2 18 08 74 22 6 2 3
